### ISSUE TITLE
arguments broken

### ISSUE THREAD

**Issue by tenox7** [tenox7/ttyplot#102]
paging @MIvanchev https://github.com/tenox7/ttyplot/pull/90 broke aruments/flags (update: for bsd/macos only), can you please fix?

**Comment by edgar-bonet** [comment]
I cannot reproduce:

```shell
$ git switch master
Already on 'master'
Your branch is up to date with 'upstream/master'.
$ git pull upstream
Already up to date.
$ git clean -dxf  # make this as clean as it gets
Removing ttyplot
$ make
cc -Wall -Wextra    ttyplot.c  `pkg-config --libs ncurses 2>/dev/null || echo '-lcurses -ltinfo'` -o ttyplot
$ ./ttyplot -v
ttyplot 1.5.1
$ ./ttyplot -h
Usage:
  ttyplot [-h] [-v] [-2] [-r] [-c plotchar] [-s scale] [-m max] [-M min] [-t title] [-u unit]

  -2 read two values and draw two plots, the second one is in reverse video
  [snip]
$ seq 1 80 | ./ttyplot -c o
# A plot is displayed with the ‘o’ character.
$ lsb_release -ds
Ubuntu 22.04.3 LTS
```

**Comment by hartwork** [comment]
@tenox7 can you define broken and share more about the environment? E.g. is it a box with ancient broken getopt by any chance?

**Comment by MIvanchev** [comment]
It could be that `optind` needs to be reset to 1 instead of 0.

**Comment by MIvanchev** [comment]
@tenox7 (and everyone else): Could you check if this MR works for you? If yes, I also incremented the version numbers for a hotfix release.

**Comment by tenox7** [comment]
haha, no the issue is on latest freebsd and macos; build from head master and ran with -2, still only shows one chart, -c and others do not work

**Comment by tenox7** [comment]
tried https://github.com/MIvanchev/ttyplot/tree/issue-102-potentially-incorrect-usage still same issue, both freebsd and macos

**Comment by MIvanchev** [comment]
@tenox7 OK, I know what the issue is, stand by.

**Comment by tenox7** [comment]
thank you :)

**Comment by MIvanchev** [comment]
@tenox7 Try now? I've added some BSD and Mac logic.

**Comment by tenox7** [comment]
checking (thank you!)

**Comment by tenox7** [comment]
same repo? just did `git clone https://github.com/MIvanchev/ttyplot.git && cd ttyplot && make ttyplot && ./ttyplot -2`

linux: works
freebsd: doesnt
macos: doesnt

**Comment by MIvanchev** [comment]
git clone --branch issue-102-potentially-incorrect-usage https://github.com/MIvanchev/ttyplot.git && cd ttyplot && make ttyplot && ./ttyplot -2

**Comment by tenox7** [comment]
ok branch `remotes/origin/issue-102-potentially-incorrect-usage` works!

linux: works
macos: works
freebsd: works

thank you! please send a PR

**Comment by MIvanchev** [comment]
It's here https://github.com/tenox7/ttyplot/pull/103, good thing you test on non Linuxes :))

**Comment by tenox7** [comment]
thank you for fixing @MIvanchev much appreciated!!!!

**Comment by MIvanchev** [comment]
NP, this is a small one, much more work to be done.

**Comment by hartwork** [comment]
@fcambus any chance you could bump ttyplot to 1.5.2 to fix issue https://github.com/tenox7/ttyplot/issues/102#issuecomment-1814217694 for users of OpenBSD?

**Comment by fcambus** [comment]
> @fcambus any chance you could bump ttyplot to 1.5.2 to fix issue [#102 (comment)](https://github.com/tenox7/ttyplot/issues/102#issuecomment-1814217694) for users of OpenBSD?

I just committed an update, thanks for the heads-up!

**Comment by hartwork** [comment]
@fcambus excellent, thank you! :+1:
